EIPS/eip-7743.md

```diff
@@ -0,0 +1,120 @@
+---
+eip: 7743
```

Comment:
```suggestion
eip: 7522
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename & Eth Magicians post.